script.js

```diff
@@ -0,0 +1,41 @@
+var arr = new Array();
+function myFunction(name,breed,fee){
+  alert("Name:"+name+"\nBreed:"+breed+"\nAdoption Fee:"+fee);
+}
+function getSum(total,num) {
+  return total + Math.round(num);
+}
+function adoptFunction(item){
+  var x = document.getElementById("price").childNodes[0].nodeValue;
+  arr.push(x);
+
+  for (var i = 0; i <= arr.length; i++) {
+    total = +arr[i];
+  }
+  document.getElementById('demo').innerHTML = arr.reduce(getSum,0);
+}
+
+const container = document.getElementById('container');
+const heading = document.createElement('h1');
+heading.textContent = 'Adoptadog Blog!';
+container.appendChild(heading);
+var images = ['blog-1.jpg','blog-2.jpg','blog-3.jpg'];
+var titles = ['Travelling With Your Dog','How To Walk Multiple Dogs','Teach Your Dog To Fetch!'];
+for(let i=0;i<images.length;i++){
```

Comment:
This is good, I would encourage you to think about how you could redo this to work without the individual sub-interface containers in the html